lib/v2/tfbd/utils.js

```diff
@@ -0,0 +1,87 @@
+const cheerio = require('cheerio');
+const got = require('@/utils/got');
+const { art } = require('@/utils/render');
+const path = require('path');
+const url = require('url');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+const base = 'http://tfbd.ccf.org.cn';
+
+const urlBase = (caty, id) => url.resolve(base, `/tfbd/${caty}/${id}/`);
```

Comment:
`url.resolve()` [is legacy](https://nodejs.org/api/url.html#urlparseurlstring-parsequerystring-slashesdenotehost). Please use WHATWG URL API instead.